tools/modules/functions/interface_checklist.sh

```
@@ -0,0 +1,114 @@
+module_options+=(
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)